help/C/Help_ch_Transactions.xml

```diff
@@ -899,180 +905,543 @@ Author:
-              the row containing that account name and edit the name in the dialog box provided. Click Next when 
-              you have finished making changes, and proceed through a similar category matching process. QIF 
-              income and expense categories import as GnuCash income and expense accounts <!--(see section 3.1 FIXME for 
-              more on this).--> Make changes to these account names if necessary, and click Next to continue.
-        </para>
-        <note><para>If you are not sure what changes are needed, it is safe to
-              accept the GnuCash account names. It is easy to edit the accounts later if you find you need to make
-              a change.
-        </para></note>
-        <para>From the drop-down list, select a standard currency to be used for the imported accounts and click <guibutton>Next</guibutton>
-              to continue. If you have stocks, mutual funds, or other securities, you will be prompted for additional
-              information. The importer dialog will ask for the exchange or listing (i.e. Nasdaq), the security's 
-              full name, and the ticker symbol. If you do not have this information handy, you can edit the account
-              information later, once the import is complete. Click <guibutton>Next</guibutton> button to continue.
-        </para>
-      </sect3>
-      <sect3>
-        <title>Tell GnuCash to import the data.</title>
-        <para>The last step is the import. Once you have verified your account names and investment information, 
-              click <guibutton>Finish</guibutton> button in the Update your GnuCash accounts page to complete the import process. Depending 
-              upon the size of your file, the import might take a few minutes to complete, so a progress bar 
-              displays the percentage finished. When the import process is complete, GnuCash will return you 
+        <screenshot id="ImportQIFDruid">
+        <mediaobject>
+          <imageobject role="html">
+            <imagedata fileref="figures/Help_Import_QIF_Druid.png" format="PNG" srccredit="David Cousens" width="510"></imagedata>
+          </imageobject>
+          <imageobject role="fo">
+                      <imagedata fileref="figures/Help_Import_QIF_Druid.png" format="PNG" srccredit="David Cousens"></imagedata>
+          </imageobject>
+          <textobject>
+            <phrase>Import QIF files dialog window.</phrase>
+          </textobject>
+          <caption><emphasis role="bold">Import QIF files dialog window.</emphasis></caption>
+        </mediaobject>
+        </screenshot>
+        <para>On most panes the dialog has three buttons on the right for navigating through the dialog steps shown in the left pane of the dialog:
+          <itemizedlist>
+            <listitem>
+              <guibutton><emphasis role="bold">Cancel</emphasis></guibutton>
```

Comment:
Nitpick: If you think, we really need the emphasis, I would prefer `<emphasis role="bold"><guibutton>Cancel</guibutton></emphasis>` because the emphasis is not part of the gui element. Search-and-replace operations will be much easier. That has several advantages. 

1. Imagine GTK renames `<guibutton>Cancel</guibutton>` to `<guibutton>Abort</guibutton>`
2. There is https://bugs.gnucash.org/show_bug.cgi?id=633673. We probably want convert it then from
```
<itemizedlist>
   <listitem>
      <emphasis role="bold"><guibutton>Cancel</guibutton></emphasis>
      <para>Exit the dialog;</para>
   </listitem>...
```
to
```
<variablelist>
   <varlistentry>
      <term><guibutton>Cancel</guibutton></term>
      <listitem><para>Exit the dialog;</para></listitem>
   </varlistentry> ...
```

Reply:
Point taken on the <emhasis role="bold". outside the <guibutton> type tags. Will find and change.

I agree with Christian Marchi's comment re the <variablelist>. I tried using it in a number of places and found the formatting extremely confusing and the meaning less clear.